learning/features/math/vectors_advanced.rst

```diff
@@ -169,7 +169,7 @@ Some examples of planes
 Here is a simple example of what planes are useful for. Imagine you have
 a `convex <http://www.mathsisfun.com/definitions/convex.html>`__
 polygon. For example, a rectangle, a trapezoid, a triangle, or just any
```

Comment:
I think I would change this to be something like "...polygon, such as a triangle, rectangle, or trapezoid."

Reply:
I tried to be as non-intrusive as I could. I only edited the typos and mistakes.

Reply:
I understand. The majority of my own contributions to this repository are non-intrusive spelling and grammar corrections!

I should have added this comment on the line below to be more clear: As you are changing this sentence already, I think it would work just as well if you simplified it further and merged it into the previous sentence. No need to worry about it getting accepted, as I already have approved the change.

Reply:
@amirea Please let us know if you are interested in making any further refinements to this sentence. If not, this is ready to merge.

Reply:
It's good. And also if it's OK with you, please merge it.